TehraniFarzam/HW6/06.1_PRACTICAL_TASKS-2.py

```diff
@@ -0,0 +1,10 @@
+login = "First"
```

Comment:
good job
look at this version of code
user_name = input('Hello, please input your Log in:')

while user_name != 'First':
    user_name = input('Error: wrong username, please try one more time. Username:')
else:
    print('Greeting. Access granted!!!', user_name)

Reply:
Lovely !!! That is very simple and functions as expected. I have already added this script to my local branch. Thank you)